nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -151,6 +151,7 @@
         <dependency>
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
+            <version>1.15</version>
```

Comment:
Is this specific version necessary with the version already specified in the root Maven configuration?

Reply:
IIRC it was. I ripped out ALL of them and 4 modules still required it. IIRC, it was due to test vs compile scoping issues with `<dependencyManagement>` in the root pom.

Reply:
Thanks, that makes some sense.